pbs_light.go

```diff
@@ -838,12 +848,14 @@ func serve(cfg *config.Configuration) error {
 		glog.Fatalf("Failed to create the amp endpoint handler. %v", err)
 	}
 
+	syncers := usersyncers.NewSyncerMap(cfg)
```

Comment:
The main `BidderMap` is in _openrtb2_ext_ package. I know `exchanges` is from old code but wouldn't it be better if we could have a single source for all bidders?

Reply:
Definitely. I'm guilty of ignoring legacy code, but... this isn't much effort, and you're right that it's valuable.

Reply:
Actually... there is a problem with this. The `exchange` includes `districtm`, which is an appnexus alias. Alias support is added to `openrtb_ext` in #296... but I don't see any value in hardcoding aliases into PBS, since they can be passed in dynamically.

So... there actually _isn't_ a single source of truth here. The `BidderMap` is a source of truth for `Usersyncer` and `Bidder` impls... but the `exchange` doesn't fit into that.

Reply:
Yes, single source excluding `exchanges`. Perfect! :smile:

Reply:
yeah... in that case, there is. The `TestSyncers` test inside `usersync/usersync_test.go` makes sure that the Usersyncer map stays up to date with the `BidderMap`, and the `TestNewExchange` test inside `exchange/exchange_test.go` makes sure that the OpenRTB Exchange stays in sync with it.

Nothing is watching out for the legacy exchange, though.

Reply:
True the test accomplishes it but I believe the code should be made fail-proof regardless of a test (at least we should try to as much as we can). It would need a switch-case in the `NewSyncerMap()` that iterates through the BidderMap. 

I'll leave it to you to decide if it's worth it. :)

Reply:
Don't think so... that just replaces a `map` declaration with a `switch` statement. We'd still have to update two places in the code whenever we add a new Bidder, so it's still definitely a point of failure (barring the unit tests)